jetty-websocket/websocket-javax-server/src/main/java/org/eclipse/jetty/websocket/javax/server/config/JavaxWebSocketServletContainerInitializer.java

```diff
@@ -148,7 +148,7 @@ public static void configure(ServletContextHandler context, Configurator configu
      * @param context the context to work with
      * @return the default {@link ServerContainer} for this context
      */
-    public static JavaxWebSocketServerContainer initialize(ServletContextHandler context)
+    public static ServerContainer initialize(ServletContextHandler context)
```

Comment:
This change fixes the JPMS issue.
Not sure if we really need this public as it is only used just below, but have left it public for now.

Reply:
The method needs to be `public` because it is how embedded applications such as e.g. CometD will initialize WebSocket.

Reply:
> This method SHOULD NOT BE CALLED by users of Jetty.

I think embedded applications should really be using 
`public static void configure(ServletContextHandler context, Configurator configurator)`